### DISCUSSION TITLE
Does the Symfony ecosystem have anything similar to Laravel Sail?

### DISCUSSION THREAD

**Discussion by abunch** [symfony/symfony#42413] [Q&A] [answered]
Just like the subject says, does the Symfony ecosystem have anything similar to Laravel Sail?

**Comment by monteiro** [comment, marked as answer]
Symfony binary has docker integration: https://symfony.com/download

**Reply by javiereguiluz** [reply]
@abunch reading Laravel Sail docs it looks like a small utility to use Docker without having to learn how to use Docker. Symfony doesn't have anything like that ... because it provides direct integration with Docker in the Symfony CLI tool that you download from https://symfony.com/download

Even if it may take a bit longer to start using it, all the things you'll learn will help you everywhere, because you are learning standard Docker, which is used everywhere, instead of a tool specific to some framework.

**Reply by abunch** [reply]
Thanks @javiereguiluz and @monteiro  - I'm familiar with Docker, however, we're deploying a solution to a team that's less familiar.  Rather than educating I was looking for a way to allow them to spin up and work without as much up-front education.  You're correct though, learning the underlying tech will help more in the long run and is a better solution.

Thanks!
